app/main.go

```diff
@@ -96,9 +95,38 @@ func main() {
 		log.Fatalf("Invalid flags: %v", err)
 	}
 
-	http.HandleFunc("/", index)
-	http.HandleFunc("/sync", sync)
-	http.HandleFunc("/cleanup", cleanup)
+	config := tsbridge.NewConfig(&tsbridge.ConfigOptions{
+		Filename:                 *metricConfig,
+		MinPointAge:              *minPointAge,
+		CounterResetInterval:     *counterResetInterval,
+		SDLookBackInterval:       *sdLookBackInterval,
+		SDInternalMetricsProject: *sdInternalMetricsProject,
+		UpdateParallelism:        *updateParallelism,
+		UpdateTimeout:            *updateTimeout,
+		EnableStatusPage:         *enableStatusPage,
+		StorageEngine:            *storageEngine,
+		SyncPeriod:               *syncPeriod,
+	})
+
+	h := web.NewHandler(config)
+
+	http.HandleFunc("/", h.Index)
+	http.HandleFunc("/sync", h.Sync)
+	http.HandleFunc("/cleanup", h.Cleanup)
+
+	// Run a cleanup on startup
+	log.Debugf("Performing startup cleanup...")
+	if err := tasks.Cleanup(context.Background(), config); err != nil {
+		log.Fatalf("error running the Cleanup() routine: %v", err)
+	}
+
+	// Run a sync loop for standalone use
+	// TODO(temikus): refactor this to run exactly every SyncPeriod and skip sync if one is already active
+	if !env.IsAppEngine() {
+		log.Debug("Running outside of appengine, starting up a sync loop...")
+		ctx, cancel := context.WithCancel(context.Background())
+		go syncLoop(cancel, config, ctx)
```

Comment:
readability: convention is to always pass context first.

`go syncLoop(ctx, cancel, config)`

Reply:
👍 Good to know, fixed!